std_msgs/Header header

geometry_msgs/Twist     pose
geometry_msgs/Twist     twist
geometry_msgs/Twist     wrench

geometry_msgs/Twist     desiredPose
geometry_msgs/Twist     desiredTwist
geometry_msgs/Twist     desiredWrench

geometry_msgs/Twist     estimatedWrench
geometry_msgs/Twist     propellersInput

geometry_msgs/Twist     error_pose
geometry_msgs/Twist     error_twist
geometry_msgs/Twist     error_wrench